lib/fastlane/plugin/maestro_orchestration/actions/maestro_orchestration_api_request_action.rb

```diff
@@ -63,28 +67,13 @@ def self.available_options
         ]
       end
 
-      def self.folder_path_option
+      def self.s3_path_option
         FastlaneCore::ConfigItem.new(
-          key: :folder_path,
-          env_name: "MAESTRO_SCREENSHOTS_FOLDER_PATH",
-          description: "Path to the folder to be uploaded to S3",
-          default_value: File.expand_path("../../.maestro/screenshots", FastlaneCore::Helper.fastlane_enabled_folder_path),
-          optional: false,
-          verify_block: proc do |value|
-            UI.user_error!("You must provide a valid folder path using the `folder_path` parameter.") unless value && !value.strip.empty?
-          end
-        )
-      end
-
-      def self.bucket_option
-        FastlaneCore::ConfigItem.new(
-          key: :bucket,
-          env_name: "MAESTRO_SCREENSHOTS_S3_BUCKET",
-          description: "The S3 bucket name where files will be uploaded",
-          optional: false,
-          verify_block: proc do |value|
-            UI.user_error!("You must provide a valid bucket name using the `bucket` parameter.") unless value && !value.strip.empty?
-          end
+          key: :s3_path,
+          env_name: "MAESTRO_SCREENSHOTS_S3_PATH",
+          description: "The base S3 path where files will be uploaded",
```

Comment:
Should the documentation include an example to suggest inclusion of the bucket as well?